Doppler.Sap/Factory/ISapServiceSettingsFactory.cs

```diff
@@ -0,0 +1,10 @@
+using Doppler.Sap.Enums;
+using Doppler.Sap.Models;
+
+namespace Doppler.Sap.Factory
+{
+    public interface ISapServiceSettingsFactory
+    {
+        ISapTaskHandler CreateHandler(string countryCode);
```

Comment:
This method makes a `new` CreateHandler sentence, so can return a concrete class SapTaskHandler

Reply:
We need the interface for the unit tests